<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarehouseSystemProductWidget</class>
 <widget class="QWidget" name="WarehouseSystemProductWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Продукция | ООО &quot;Нинель&quot;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frMain">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="tvProducts"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pbAddProduct">
        <property name="text">
         <string>Добавить продукцию</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
